include/openenclave/internal/sgxtypes.h

```diff
@@ -572,7 +572,7 @@ oe_thread_data_t* oe_get_thread_data(void);
 
 #define OE_THREAD_LOCAL_SPACE (OE_PAGE_SIZE)
 
-#define OE_THREAD_SPECIFIC_DATA_SIZE (3840)
+#define OE_THREAD_SPECIFIC_DATA_SIZE (3824)
```

Comment:
Can we have more information regarding this number?

Reply:
I documented how/why OE_THREAD_SPECIFIC_DATA_SIZE is computed the way it is. And also added an assertion.